### PR TITLE
Updated Jackett to new version to stop breaking change in Jackett

### PR DESCRIPTION
**Pull request by GazzaHazza**
_Motivation: Update version to a working version to users can access torrentday
_Linked issues: https://github.com/SynoCommunity/spksrc/issues/2559

### Checklist
- [] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by cytec** [comment]
LGTM @KaraokeStu you up to updating/publish the SPK? im still on a 32bit env with my current setup so i'd rather not do it

File: spk/jackett/Makefile [hunk 1]
```diff
@@ -13,6 +13,7 @@ ADMIN_PORT = 9117
 RELOAD_UI = yes
 DISPLAY_NAME = Jackett
 BETA = 1
+CHANEGLOG = Updated to version 0.7.806
```
Comment (Dr-Bean) [inline spk/jackett/Makefile]: Spelling incorrect; `CHANGELOG`

**Comment by Dr-Bean** [comment]
Build started, thanks for the contribution :)

**Comment by GazzaHazza** [comment]
No worries! thanks for the help getting up and running!

**Comment by Dr-Bean** [comment]
jackett 0.7.806-3 has been published on the repository.

**Pull request merged**